examples/dump.php

```diff
@@ -27,7 +25,7 @@
     echo 'Received entry headers:' . PHP_EOL;
     var_dump($header);
 
-    BufferedSink::createPromise($file)->then(function ($contents) {
+    $file->on('data', function ($contents) {
```

Comment:
This will now print this message for each chunk of data whereas the old version buffered the whole file. Possible alternative: https://github.com/reactphp/promise-stream#buffer